### PR TITLE
Apply-discount-code-placeholder-get-cut ::Apply discount code placeho…

### PR DESCRIPTION
**Pull request by ajay2jcommerce**
…lder gets cut in Tab portriat view on cart page

### Fixed Issues (if relevant)

1. fixed issue #20278: Apply discount code placeholder gets cut in Tab portriat view on cart page 

### Manual testing scenarios (*)

1. Add product in shopping cart
2. Goto cart page
3. Click on "Apply Discount Code"

### Contribution checklist (*)
 - [ ] Pull request has a meaningful description of its purpose
 - [ ] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with unit/integration tests (if applicable)
 - [ ] All automated tests passed successfully (all builds on Travis CI are green)

### CODE REVIEW

**Comment by magento-engcom-team** [comment]
Hi @ajay2jcommerce. Thank you for your contribution 
Here is some useful tips how you can test your changes using Magento test environment.
Add the comment under your pull request to deploy test or vanilla Magento instance:
- `@magento-engcom-team give me test instance` - deploy test instance based on PR changes
- `@magento-engcom-team give me 2.3-develop instance` - deploy vanilla Magento instance

For more details, please, review the [Magento Contributor Assistant](https://devdocs.magento.com/guides/v2.2/contributor-guide/contributing.html#contributor-assist) documentation

File: app/design/frontend/Magento/luma/Magento_Checkout/web/css/source/module/_cart.less [hunk 1]
```diff
@@ -694,6 +694,9 @@
                     position: static;
                 }
             }
+            &.discount{
+                width:auto;
```
Comment (VladimirZaets) [inline app/design/frontend/Magento/luma/Magento_Checkout/web/css/source/module/_cart.less]: Hi @ajay2jcommerce. Thanks for collaboration. Can you explain why do you use `width:auto` if its default value of width property?

Comment (ajay2jcommerce) [reply app/design/frontend/Magento/luma/Magento_Checkout/web/css/source/module/_cart.less]: @VladimirZaets 

In default magento 2.3.0 when using width: 48% issue still getting generate. When use width:auto issue get fixed reason using that property.

**Review by VladimirZaets — APPROVED** [review]

File: app/design/frontend/Magento/luma/Magento_Checkout/web/css/source/module/_cart.less (hunk 1 above)
Comment (VladimirZaets) [reply app/design/frontend/Magento/luma/Magento_Checkout/web/css/source/module/_cart.less]: @ajay2jcommerce ok, thanks

**Comment by magento-engcom-team** [comment]
Hi @VladimirZaets, thank you for the review.
[ENGCOM-3901](https://jira.corp.magento.com/browse/ENGCOM-3901) has been created to process this Pull Request

**Comment by magento-engcom-team** [comment]
Hi @ajay2jcommerce. Thank you for your contribution. 
We will aim to release these changes as part of 2.3.1. 
Please check the release notes for final confirmation.

**Pull request merged**